src/main/scala/com/lightbend/conductr/sbt/LagomConductRPlugin.scala

```diff
@@ -0,0 +1,41 @@
+package com.lightbend.conductr.sbt
+
+import sbt._
+import sbt.Keys._
+
+import com.typesafe.sbt.SbtNativePackager
+
+/**
+ * Adds Lagom concerns to the ConductR plugin
+ */
+object LagomConductRPlugin extends AutoPlugin {
+
+  import ConductrPlugin.autoImport._
+  import ConductrKeys._
+
+  import LagomBundlePlugin.autoImport._
+
+  import SbtNativePackager.autoImport._
+  import NativePackagerKeys._
+
+  override def requires = ConductrPlugin && LagomBundlePlugin
+
+  override def trigger = allRequirements
+
+  override def buildSettings: Seq[Setting[_]] =
+    List(
+      installationScriptContents ++= installationScriptContentsTask.value
```

Comment:
The same comment I made [here](https://github.com/typesafehub/sbt-conductr/pull/156/files#r64393921) may apply